src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/BuildTest.cs

```diff
@@ -653,7 +653,7 @@ public void BuildAotApplication (string supportedAbis, bool enableLLVM, bool exp
 				// Set //uses-sdk/@android:minSdkVersion so that LLVM uses the right libc.so
 				proj.AndroidManifest = $@"<?xml version=""1.0"" encoding=""utf-8""?>
 <manifest xmlns:android=""http://schemas.android.com/apk/res/android"" android:versionCode=""1"" android:versionName=""1.0"" package=""{proj.PackageName}"">
-	<uses-sdk android:minSdkVersion=""10"" />
+	<uses-sdk android:minSdkVersion=""16"" />
```

Comment:
I think this should be explained in the commit message.

Reply:
Done